### PR TITLE
Sdi 168 calendar screen   mobile UI & content changes

### PR DESCRIPTION
**Pull request by steverendell**
Redesign of the mobile calendar

### CODE REVIEW

File: server/views/pages/index.ejs [hunk 1]
```diff
@@ -38,7 +38,7 @@
                       Establishment
                     </li>
                     <li>
-                      Personal mobile number, or personal email address (or both) – we will then use these to send you notifications
+                      Personal email address – we will then use this to send you notifications
```
Comment (steverendell) [inline server/views/pages/index.ejs:41]: Should have been changed along with the banner about the removal of SMS for notifications

File: helm_deploy/values-dev.yaml [hunk 2]
```diff
@@ -11,7 +11,6 @@ generic-service:
   env:
     MAINTENANCE_START: "2022-04-21T14:00:00"
     MAINTENANCE_END: "2022-04-22T13:00:00"
-    REGIONS: "https://api.check-my-diary-dev.hmpps.dsd.io/api/health"
```
Comment (steverendell) [inline helm_deploy/values-dev.yaml:14]: unused

File: server/helpers/utilities.js [hunk 3]
```diff
@@ -105,44 +107,82 @@ const configureCalendar = (data, startDate = null) => {
...
+
+    if (displayType) {
+      const specialActivityColour = (!isFullDay && displayType.endsWith('START') && fullDayMatch(activity)) || ''
+      const showNightHr = nightFinish && displayType === 'NIGHT_START'
+      nightFinish = displayType === 'NIGHT_FINISH'
+      const durationColour = nightFinish
+        ? 'night_finish'
+        : (!isFullDay && displayType.endsWith('FINISH') && fullDayMatch(activity)) || ''
+      const activityText = `${getTaskText(displayType)}: ${moment(displayTypeTime).format('HH:mm')}`
+      Object.assign(detail, {
```
Comment (petergphillips) [inline server/helpers/utilities.js:165]: not very nice to reassign `detail` each time like this.  Could you use `reduce` instead to end up with a new `details` object instead?  That way you wouldn't need a `nightFinish` variable either as you could pass the state through too...

Comment (steverendell) [reply server/helpers/utilities.js:165]: Just kept the existing approach, but yes, can do in the next iteration 👍

**Pull request merged**